howtos/manage_cpp_standard.rst

```diff
@@ -84,11 +82,17 @@ When the ``cppstd`` setting is declared in the recipe and the consumer specifies
 Package compatibility
 ---------------------
 
-By default Conan will detect the default standard of your compiler to not generate different binary packages.
-For example, you already built some ``gcc > 6.1`` packages, where the default std is ``gnu14``.
-If you introduce the ``cppstd`` setting in your recipes and specify the ``gnu14`` value, Conan won't generate
+By default Conan will use the default standard for your compiler, so it won't generate different
```

Comment:
Idem. Not accurate. You can say that when no value is specified, Conan will compute by default the same package_id corresponding to the default std of the compiler, so when you explicit the std value for the default of the compiler, the id will be the same as no specifying it.

Reply:
That's right, and very important